packages/components/pie-cookie-banner/src/defs.ts

```diff
@@ -148,7 +180,37 @@ export type DefaultProps = ComponentDefaultProps<CookieBannerProps>;
 export const defaultProps: DefaultProps = {
     hasPrimaryActionsOnly: false,
     defaultPreferences: {},
-    locale: defaultLocale,
+    country: Country.GREAT_BRITAIN,
+    language: Language.ENGLISH,
     cookieStatementLink: '',
     cookieTechnologiesLink: '',
 };
+
+export const getDefaultLanguageForCountry = (country: string): string => {
```

Comment:
Here is what I was thinking could be preferable to a large switch statement:

```
export const Language = {
    BULGARIAN: 'bg',
    CATALAN: 'ca',
    DANISH: 'da',
    DUTCH: 'nl',
    ENGLISH: 'en',
    FRENCH: 'fr',
    GERMAN: 'de',
    HEBREW: 'he',
    ITALIAN: 'it',
    POLISH: 'pl',
    SLOVAK: 'sk',
    SPANISH: 'es',
} as const;

export const Country = {
    BULGARIA: 'bg',
    DENMARK: 'dk',
    FRANCE: 'fr',
    GERMANY: 'de',
    GREAT_BRITAIN: 'gb',
    ISRAEL: 'il',
    ITALY: 'it',
    NETHERLANDS: 'nl',
    POLAND: 'pl',
    SLOVAKIA: 'sk',
    SPAIN: 'es',
} as const;

const languageMap = new Map<string, string>([
    [Country.GREAT_BRITAIN, Language.ENGLISH],
    [Country.FRANCE, Language.FRENCH],
    [Country.DENMARK, Language.DANISH],
    [Country.SPAIN, Language.SPANISH],
    [Country.ITALY, Language.ITALIAN],
    [Country.NETHERLANDS, Language.DUTCH],
    [Country.POLAND, Language.POLISH],
    [Country.SLOVAKIA, Language.SLOVAK],
    [Country.BULGARIA, Language.BULGARIAN],
    [Country.GERMANY, Language.GERMAN],
    [Country.ISRAEL, Language.HEBREW],
]);

export const getDefaultLanguageForCountry = (country: string): string => {
    return languageMap.get(country) || defaultProps.language;
};
```

Reply:
Will that not default **all** countries to the same default language (_en_)?

Reply:
if `defaultProps.language` is set to `en` then yes 👍